lib/validator/configValidator.py

```diff
@@ -0,0 +1,182 @@
+import os
+import json
+import jsonschema
+from lib.common.logConfig import get_logger
+
+logger = get_logger(__name__)
+
+
+class ConfigValidator(object):
+    """
+    Config Validator.
+    Base on JSON Schema.
+    """
+    CONFIG_ROOT = 'configs'
+    CONFIG_EXT = '.json'
+    SCHEMA_EXT = '.schema'
+
+    @staticmethod
+    def load_json_file(fp):
```

Comment:
There is one load_json_file function implemented in commonUtil already, you should use that instead of writing a new one.